### PR TITLE
Remove remaining uses of `LocalizePF2e`

### PR DESCRIPTION
**Pull request by In3luki**

### CODE REVIEW

File: src/module/system/localize.ts [hunk 1]
```diff
@@ -8,20 +7,4 @@ type TranslationsPF2e = Record<string, TranslationDictionaryValue> &
...
-        }
-        if (this._translations === undefined) {
-            this._translations = mergeObject(game.i18n._fallback, game.i18n.translations, {
-                enforceTypes: true,
-            }) as TranslationsPF2e;
-        }
-        return this._translations;
-    }
-}
+export { TranslationsPF2e };
```
Comment (stwlam) [inline src/module/system/localize.ts:10]: Could this be moved to an existing file?

Comment (In3luki) [reply src/module/system/localize.ts:10]: It's not really used anywhere but eslint complains if one of the json files isn't imported. 

image

Comment (stwlam) [reply src/module/system/localize.ts:10]: If `TranslationsPF2e` isn't used anywhere, can be kill the whole file?

Comment (In3luki) [reply src/module/system/localize.ts:10]: Yeah, if I can figure out how to appease eslint,

Comment (stwlam) [reply src/module/system/localize.ts:10]: Push it up with eslint complaining

Comment (In3luki) [reply src/module/system/localize.ts]: Done

**Pull request merged**